src/applications/mhv/medications/api/rxApi.js

```diff
@@ -0,0 +1,38 @@
+import environment from 'platform/utilities/environment';
+import { apiRequest } from 'platform/utilities/api';
+import prescriptions from '../tests/fixtures/presciptions.json';
+
+const apiBasePath = `${environment.API_URL}/my_health/v1`;
```

Comment:
```suggestion
const apiBasePath = `/my_health/v1`;
```

The `apiRequest` helper [provides the API url itself](https://github.com/department-of-veterans-affairs/vets-website/blob/0228c98e32b93ca14812495c7550c95f60b8ff3b/src/platform/utilities/api/index.js#L93), so it may not be necessary to pass it in. Seems like you'd also have to send `optionalSettings.apiVersion` as the version is not at the root of the path... Kind of a matter of taste really 🙂